lib/huk_load/huk_load.c

```diff
@@ -0,0 +1,102 @@
+#include <sys/printk.h>
```

Comment:
Nothing wrong here, but I think we should use `lib/hw_unique_key` for path.

Or could this be first step into key management in general, where the Hardware Unique Key is just one key among others, and thus generalize the folder name already, @frkv any thoughts on this ?

Reply:
This lib is very specific to the nRF52840 and it is only relevant for loading Hardware Unique Key.  I don't think that is relevant with other key managment/derivation. So hw_unique_key seems appropriate I think.